tags.go

```diff
@@ -0,0 +1,89 @@
+package brokerTags
+
+import (
+	"time"
+)
+
+type TagManager struct {
+	broker          string
+	cfClientWrapper CFClientWrapper
+}
+
+func NewTagManager() (*TagManager, error) {
+	cfClientWrapper, err := NewCFClientWrapper()
+	if err != nil {
+		return nil, err
+	}
+	return &TagManager{
+		cfClientWrapper: cfClientWrapper,
+	}, nil
+}
+
+func (t *TagManager) GenerateTags(
+	action string,
+	serviceGUID string,
+	servicePlanGUID string,
+	organizationGUID string,
+	spaceGUID string,
+	instanceGUID string,
+) (map[string]string, error) {
+	tags := make(map[string]string)
+
+	tags["client"] = "Cloud Foundry"
+
+	tags["broker"] = t.broker
+
+	tags[action+" at"] = time.Now().Format(time.RFC822Z)
```

Comment:
The specific callout to `time.RFC822Z` is interesting enough to warrant a comment — why that particular format?

Reply:
The real reason is that I copied this piece of code from https://github.com/cloud-gov/s3-broker/blob/main/broker/broker.go#L520

That being said, [of the available time formats](https://pkg.go.dev/time#pkg-constants), this one seems the easiest to read at a glance while also giving you everything you need to know, including timezone information.